app/webpack/lifelists/show/components/details_view.jsx

```diff
@@ -122,8 +123,7 @@ const DetailsView = ( {
     }
   } else if ( lifelist.detailsView === "species" ) {
     title = I18n.t( "views.lifelists.all_species" );
-    let count = lifelist.detailsTaxon
-      ? lifelist.detailsTaxon.descendantCount : lifelist.leavesCount;
+    let count = lifelist.detailsTaxon ? _.size( filteredNodes( lifelist ) ) : lifelist.leavesCount;
```

Comment:
We still want the count to reflect the chosen rank when `lifelist.detailsTaxon` is blank. To see the problem,

1. Go to /lifelists/:username where :username is that of a user that has observations in several kingdoms
1. Change the rank filter to "Show: Kingdoms"
1. "Observed Kingdoms" shows the rank you chose, but the count is still the leaf count

Reply:
Agreed. I think we can skip the ternary and just go with `_.size( filteredNodes( lifelist ) )`